packaging/test_irods_client_rest_cpp.py

```diff
@@ -190,6 +191,300 @@ def test_put_configuration(self):
         os.remove(file1)
         os.remove(file2)
 
+    def test_meta(self):
```

Comment:
A function this large is going to be difficult to maintain.

It would help the reader if the function were split up into smaller chunks or multiple tests so that it's easier to digest.

Reply:
I think splitting this into multiple tests may be a good idea as well (one each for users, collections, data objects, and resources).